meetings/2022-05-09.md

```diff
@@ -0,0 +1,182 @@
+    * SC: authors need to validate the test reports to have the proper weight.
+    * AB: sometimes an issue with the specification was raised.
+    * SC: None of that rules this out.
+    * YB: how do we get authors to approve a test?
+    * SC: Open a PR?
+    * YB: ... how to get approval? can we do things without approval? Maybe publish parts of the report with a marker that 'this has not been validated by the author?'
+    * SC: The focus lies in what is authoritative and approved. If we have a way where we can keep the tests and the reports under review, separately. At least we are clear about what the authors have reviewed. Most of the specs are not final, authors don't always have time to dip into the tests.
+    * SC: Take test solid-oidc, has it's own test suite. The author of the spec and whoever is writing the tests, should agree that the tests are valid, if they are to be published on solidproject.org.
+    * SC: We need to agree on what the charter aims on. Summaries and reports may need to have specific machine readable information, things like that.
+    * MJ: So the tests in the specification tests are all approved by the authors? Are there sets of approved tests? Together, they cover all the tests that we know of. We hope that in time, the number of approved tests should grow. Will publish elsewhere for now.
+    * SC: would like issues on what the requirement are and put those into the charter.
+    * MJ: Not interested in a charter with the current 3rd and 4th point. Interested in a charter with how we operate.
+    * MJ: We hope that the authors will approve the tests over time, until then?
+    * AB: We need a way to collaborate with the spec authors to have validation.
+    * SC: Need some common agreement on how this will work.
+    * MJ: Want to run all the tests, approved or not.
+    * SC: The group will not publish without checks, will not publish random things.
+    * MJ: Question is, what will we do while we wait until the authors have approved the tests?
+    * SC: If we can capture how the process looks like for an author to approve a test, that should help us forward.
+    * MJ: Wasn't that the aim for the harness, so that the tests are easier to review? Having more tests creates a more level playing field for new contenders.
+    * SC: The report should not be coupled to the tooling. doesn't matter what tool or test generated it. Should only care about the data at that point, but that data should have gone through checkpoints to make sure that it is valid.
+    * MJ: Jest based tests and RDF based tests that Pete started work on should be in scope for the test-suite-panel.
+    * SC: what is the common data format for the reports, machine / human readable. Should have the name of the project, lists all the tests and outcomes.
+
+ACTION SC: Make issues for the charter proposals. Needs to leave room for unapproved reports / tests.
+
+  * Software maintainers give consent for reports to be published.
```

Comment:
We should probably remove this line from the minutes because it was on the agenda but we didn't end up discussing it.

Reply:
https://github.com/solid/test-suite-panel/pull/1/commits/7f4a73433a4f6aa28fbb7e6fb75de741f90f7315